Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -92,16 +93,24 @@ final class PaymentViewController: UIViewController {
 
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
-
-    // TODO: Remove when the new header view is added
-    navigationController?.isNavigationBarHidden = false
+    navigationController?.setNavigationBarHidden(false, animated: animated)
     setUpKeyboard()
   }
 
   required init?(coder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
 
+  private func setupNavigationBar() {
+    customizeNavigationBarAppearance(color: viewModel.paymentFormStyle?.headerView.backgroundColor ?? .white)
```

Comment:
will our form support dark mode if we default to white?

Reply:
We are not supporting dark mode yet